Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```diff
@@ -0,0 +1,72 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+fn convert_years_to_days(years: BigUint) -> BigUint {
+    NUMBER_OF_DAYS_IN_A_YEAR * years
+}
+
+fn convert_string_to_biguint(string: &str) -> Result<BigUint, ParseBigIntError> {
+    let result = BigUint::from_str(string.trim())?;
+
+    Ok(result)
+}
+
+fn get_input() -> io::Result<String> {
+    let mut buffer = String::new();
+
+    let stdin = io::stdin();
+
+    stdin.read_line(&mut buffer)?;
+
+    Ok(buffer)
```

Comment:
```suggestion
    let mut buffer = String::new();
    let stdin = io::stdin();
    stdin.read_line(&mut buffer)?;
    Ok(buffer)
```
Too many empty lines.

Reply:
Shouldn't I just return the Result here as well?

I did this according to the documentation.